tests/integration/files/file/base/issue-49273.sls

```diff
@@ -0,0 +1,15 @@
+barrier:
+  test.nop
+  
+{%- for x in [1,2,3,4,5,6,7,8,9,10] %}
```

Comment:
```suggestion
{%- for x in range(1, 10) %}
```

Reply:
@s0undt3ch Agreed.  It would make those tests more manageable.  Should we write state files out to a temp location and then clean up after the test runs, or use some other method?

Reply:
Look at how I did it in https://github.com/saltstack/salt/blob/master/tests/integration/states/test_pkgrepo.py#L356

It takes care of proper placement and deletion in the end.

Reply:
I like that.  Updated the PR with similar changes.  Thanks!